app/services/automation_rules/action_service.rb

```diff
@@ -1,53 +1,88 @@
 class AutomationRules::ActionService
-  def initialize(rule, conversation)
+  def initialize(rule, account, conversation_ids)
     @rule = rule
-    @conversation = conversation
-    @account = @conversation.account
+    @account = account
+    @conversations = Conversation.where(id: conversation_ids)
   end
 
   def perform
-    @rule.actions.each do |action, _current_index|
+    @rule.actions.each do |action|
       action = action.with_indifferent_access
-      send(action[:action_name], action[:action_params])
+      begin
+        send(action[:action_name], action[:action_params])
+      rescue StandardError => e
+        Sentry.capture_exception(e)
+      end
     end
   end
 
   private
 
+  def send_email_transcript(email)
+    @conversations.each do |conversation|
+      ConversationReplyMailer.with(account: conversation.account).conversation_transcript(conversation, email)&.deliver_later
+    end
+  end
+
+  def mute_conversation(_params)
+    @conversations.each(&:mute!)
+  end
+
+  def change_status(status)
+    @conversations.each do |conversation|
+      conversation.update_column(:status, status[0])
```

Comment:
@sojan-official We used the update_column here to avid callback, which was again causing a call to conversation_updated automation actions.